core/server/master/pom.xml

```diff
@@ -136,6 +141,12 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpclient</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-underfs-s3a</artifactId>
+      <version>2.10.0-SNAPSHOT</version>
```

Comment:
${project.version}?

Reply:
removed this one. this was added by mistake